### PR TITLE
Feature/209 full submissions overview

### PR DESCRIPTION
**Pull request by zerolab**
Adds 
- reviewer count, reviews and traffic light recommendation for each submission
- last updated logic based on activity
- some clean up

### CODE REVIEW

File: opentech/apply/review/models.py [hunk 1]
```diff
@@ -26,28 +26,28 @@ def by_reviewers(self):
     def staff_score(self):
         return self.by_staff().score()
 
-    def staff_reccomendation(self):
-        return self.by_staff().reccomendation()
+    def staff_recommendation(self):
```
Comment (todd-dembrey) [inline opentech/apply/review/models.py]: Need to update the template with these changes

File: opentech/apply/review/templatetags/review_tags.py [hunk 2]
```diff
@@ -7,17 +7,29 @@
...
+        'value': 'M',
+    },
+    NO: {
+        'color': 'red',
+        'value': 'N'
+    }
 }
 
-TRAFFIC_LIGHT_TEMPLATE = '<span class="traffic-light traffic-light__{color}"></span>'
+TRAFFIC_LIGHT_TEMPLATE = '<span class="traffic-light traffic-light__{color}">{value}</span>'
```
Comment (chris-lawton) [inline opentech/apply/review/templatetags/review_tags.py]: perfect

File: opentech/apply/review/templatetags/review_tags.py [hunk 3]
```diff
@@ -7,17 +7,29 @@
...
 
 
 @register.filter()
 def traffic_light(value):
     try:
-        return mark_safe(TRAFFIC_LIGHT_TEMPLATE.format(color=TRAFFIC_LIGHT_COLORS[value]))
+        return mark_safe(TRAFFIC_LIGHT_TEMPLATE.format(
+            color=TRAFFIC_LIGHT_COLORS[value]['color'],
+            value=TRAFFIC_LIGHT_COLORS[value]['value']
```
Comment (todd-dembrey) [inline opentech/apply/review/templatetags/review_tags.py]: `mark_safe(TRAFFIC_LIGHT_TEMPLATE.format(**TRAFFIC_LIGHT_COLORS[value])`

File: opentech/apply/review/models.py (hunk 1 above)
Comment (zerolab) [reply opentech/apply/review/models.py]: was there, got missed in a rebase. now fixed

**Comment by zerolab** [comment]
OK, ready. There is the traffic light sort bit that is outstanding but that will be a follow-up depending on the feedback from Dan

**Pull request merged**